src/RetryMiddlewareFactory.php

```diff
@@ -64,8 +63,7 @@ public static function getRetryFunctionByRange(
         return function (
             $retries,
             Request $request,
-            Response $response = null,
-            RequestException $exception = null
+            Response $response = null
```

Comment:
composer.json requires `"php": ">=7.3"` so I removed the trailing commas not supported until 8.0.0: https://www.php.net/manual/en/functions.arguments.php#example-151